maintainers/maintainer-list.nix

```diff
@@ -867,6 +867,11 @@
     github = "cko";
     name = "Christine Koppelt";
   };
+  clacke = {
```

Comment:
Please add yourself as a maintainer in a separate commit.

Like `maintainers: add clacke`.

Reply:
✓